### PR TITLE
Bandaid fix for single quotes in command args

### PR DESCRIPTION
**Pull request by sovdeeth**
### Description

Manually escapes single quotes in the generated command usage message to avoid errors when trying to use single quotes as command arguments. The fix is a bandaid as there are many more problems with command parsing/argument parsing that should be addressed: ``command test %now%`` being a good example. This can come in a later pr.

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

File: src/test/skript/tests/regressions/pull-6936-single quotes in commands.sk [hunk 1]
```diff
@@ -0,0 +1,7 @@
+command single-quotes-in-commands ["<text="success 2">"]:
+	trigger:
+		broadcast arg
+
+test "single-quotes-in-commands":
+	execute console command "single-quotes-in-commands ""success 1"""
```
Comment (Pikachu920) [inline src/test/skript/tests/regressions/pull-6936-single quotes in commands.sk:6]: so the test is considered successful if the broadcast appears?

Comment (sovdeeth) [reply src/test/skript/tests/regressions/pull-6936-single quotes in commands.sk:6]: Yes

Comment (sovdeeth) [reply src/test/skript/tests/regressions/pull-6936-single quotes in commands.sk:6]: I suppose i could figure out some assertion via the argument? Not sure how that'd work but might be doable

Comment (Pikachu920) [reply src/test/skript/tests/regressions/pull-6936-single quotes in commands.sk:6]: you could set a variable in the command and assert that it is set

Comment (sovdeeth) [reply src/test/skript/tests/regressions/pull-6936-single quotes in commands.sk:6]: good idea

File: src/test/skript/tests/regressions/pull-6936-single quotes in commands.sk [hunk 2]
```diff
@@ -1,7 +1,10 @@
 command single-quotes-in-commands ["<text="success 2">"]:
 	trigger:
-		broadcast arg
+		set {sqic::output} to arg
```
Comment (Pikachu920) [inline src/test/skript/tests/regressions/pull-6936-single quotes in commands.sk:3]: image
sqic

Comment (sovdeeth) [reply src/test/skript/tests/regressions/pull-6936-single quotes in commands.sk:3]: sqic sqic

**Review by APickledWalrus — APPROVED** [review]

File: src/main/java/ch/njol/skript/command/CommandUsage.java [hunk 3]
```diff
@@ -46,6 +46,10 @@ public class CommandUsage {
 	 */
 	public CommandUsage(@Nullable VariableString usage, String defaultUsage) {
 		if (usage == null) {
+			// Manually escape quotes. This is not a good solution, as it doesn't handle many other issues, like % in
+			// commands, but in lieu of re-writing the argument parser and command logic completely, I believe this is
+			// a decent stop-gap measure for using " in commands.
+			defaultUsage = defaultUsage.replaceAll("\"", "\"\"");
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/command/CommandUsage.java:52]: Taking another look at this, do you think either of these methods would be useful?
https://github.com/SkriptLang/Skript/blob/0e10975d55b87b742a38ed9f4b01e9cfead1269d/src/main/java/ch/njol/skript/lang/VariableString.java#L270
https://github.com/SkriptLang/Skript/blob/0e10975d55b87b742a38ed9f4b01e9cfead1269d/src/main/java/ch/njol/skript/lang/VariableString.java#L291

Comment (sovdeeth) [reply src/main/java/ch/njol/skript/command/CommandUsage.java:52]: quote() may work, i'll have to test

**Review by UnderscoreTud — APPROVED** [review]

**Pull request merged**